src/js/profile/wearable/widget/wearable/SnapListview.js

```diff
@@ -448,7 +448,7 @@
 				// finding page  and scroller
 				ui.page = utilSelector.getClosestByClass(listview, "ui-page") || document.body;
 
-				scroller = getScrollableParent(listview);
+				scroller = getScrollableParent(listview) || ui.page;
```

Comment:
I have suggestion to this code. The standard scrolling layer is the element with class "ui-scroller". This element is created during build process of page.
```
scroller = getScrollableParent(listview) ||
    **ui.page.querySelector(".ui-scroller")** ||
    ui.page;
```